src/features/home/components/HomeBodyPlaceholder.tsx

```diff
@@ -38,23 +41,37 @@ export const HomeBodyPlaceholder = () => (
 const OfferModulePlaceholder: React.FC<{ size: TileSize; numberOfTiles: number }> = ({
   size,
   numberOfTiles,
-}) => (
-  <Row>
-    <Spacer.Row numberOfSpaces={6} />
-    <View>
-      <ModuleTitlePlaceholder />
-      <Spacer.Column numberOfSpaces={4} />
-      <Row>
-        {new Array(numberOfTiles).fill(null).map((_, index) => (
-          <React.Fragment key={`placeholder-${index}`}>
-            <OfferTilePlaceholder size={size} />
-            <Spacer.Row numberOfSpaces={4} />
-          </React.Fragment>
-        ))}
-      </Row>
-    </View>
-  </Row>
-)
+}) => {
+  const data = new Array(numberOfTiles).fill(null)
+  const renderPlaceholder = useCallback(
+    ({ index }: { index: number }) => {
+      return (
+        <React.Fragment key={`placeholder-${index}`}>
+          <OfferTilePlaceholder size={size} />
+          <Spacer.Row numberOfSpaces={4} />
```

Comment:
si on garde la Flatlist, on peut utiliser son API jusqu'au bout:

```js
  const data = new Array(numberOfTiles).fill(null).map((_, index: number) => ({ key: index }))

  const renderPlaceholder = useCallback(
    ({ item }) => <OfferTilePlaceholder key={`placeholder-${item.key}`} size={size} />,
    [size]
  )

  return (
    <Row>
      <Spacer.Row numberOfSpaces={6} />
      <View>
        <ModuleTitlePlaceholder />
        <Spacer.Column numberOfSpaces={4} />
        <FlatList
          horizontal
          data={data}
          renderItem={renderPlaceholder}
          ItemSeparatorComponent={() => <Spacer.Row numberOfSpaces={4} />}
          showsHorizontalScrollIndicator={false}
        />
      </View>
    </Row>
  )
```

Reply:
mais en inspectant why-did-you-render, j'ai pas l'impression que ce soit mieux avec une Flatlist